index.ts

```diff
@@ -148,6 +149,9 @@ export class SpeechRecorder {
     if (!this.disableSecondPass && speaking && this.vadBuffer.length == this.vadBufferSize) {
       probability = await this.vad.process([].concat(...this.vadBuffer));
       speaking = probability > this.vadThreshold;
+      if (speaking && this.vadRateLimit > 0) {
```

Comment:
Does the second term always evaluate to 0 > 0 and so the condition is always false?

Reply:
oh thanks, forgot to add an option, was just testing by manually changing this value